docs/source/models/pic.rst

```diff
@@ -25,65 +26,94 @@ for multiple particle species :math:`s`.
 
 Except for normalization of constants, PIConGPU implements the governing equations in SI units.
 
-Relativistic Plasma Physics
+Vlasov--Maxwell equation
 ---------------------------
 
-The 3D3V particle-in-cell method is used to describe many-body systems such as a plasmas.
-It approximates the Vlasov--Maxwell--Equation
+The 3D3V particle-in-cell algorithm is used to describe many-body systems such as a plasmas.
+It simulates its by calculating the motion of its electrons and ions in the present electromagnetic fields with the Vlasov--Maxwell equation
 
 .. math::
    :label: VlasovMaxwell
 
-   \partial_t f_s(\mathbf{x},\mathbf{v},t) + \mathbf{v} \cdot \nabla_x f_s(\mathbf{x},\mathbf{v},t) + \frac{q_s}{m_s} \left[ \mathbf{E}(\mathbf{x},t)  + \mathbf{v} \times \mathbf{B}(\mathbf{x},t) \right] \cdot \nabla_v f_s(\mathbf{x},\mathbf{v},t) = 0
+   \partial_t f_s(\mathbf{x},\mathbf{p},t) + \mathbf{v} \cdot \nabla_x f_s(\mathbf{x},\mathbf{p},t) + q_s \left[ \mathbf{E}(\mathbf{x},t)  + \frac{\mathbf{v}}{c} \times \mathbf{B}(\mathbf{x},t) \right] \cdot \nabla_p f_s(\mathbf{x},\mathbf{p},t) = 0
 
-with :math:`f_s` as the distribution function of a particle species :math:`s`, :math:`\mathbf{x},\mathbf{v},t` as position, velocity and time and :math:`\frac{q_s}{m_s}` the charge to mass-ratio of a species.
-The momentum is related to the velocity by :math:`\mathbf{p} = \gamma m_s \mathbf{v}`.
-
-The equations of motion are given by the Lorentz force as
+with :math:`f_s` as the distribution function of a particle species :math:`s`, :math:`\mathbf{x},\mathbf{p},t` as position, momentum and time and :math:`{q_s},{m_s}` the charge and mass of a species.
+The velocity is related to the momentum by
 
 .. math::
+    \mathbf{v} = \frac{c \mathbf{p}}{\sqrt{(m_s c)^2 + \abs{p}^2}}.
 
-   \frac{\mathrm{d}}{\mathrm{d}t} \mathbf{V_s}(t) &= \frac{q_s}{m_s}  \left[ \mathbf{E}(\mathbf{X_s}(t),t) + \mathbf{V_s}(t) \times \mathbf{B}(\mathbf{X_s}(t),t) \right]\\
-   \frac{\mathrm{d}}{\mathrm{d}t} \mathbf{X_s}(t) &= \mathbf{V_s}(t) .
-
-.. attention::
 
-   TODO: write proper relativistic form
+Electro-Magnetic PIC Method
+---------------------------
 
-:math:`\mathbf{X}_s = (\mathbf x_1, \mathbf x_2, ...)_s` and :math:`\mathbf{V}_s = (\mathbf v_1, \mathbf v_2, ...)_s` are vectors of *marker* positions and velocities, respectively, which describe the ensemble of particles belonging to species :math:`s`.
+The distribution of of **particles** is described by the distribution function :math:`f_s(\mathbf{x},\mathbf{p},t)` and tracked in a continuous 6D (5D) phase space :math:`(\mathbf{r},\mathbf{p})`.
```

Comment:
I would maybe elaborate that the (5D) means 2D 3V simulations, that confused me for a second